### ISSUE TITLE
Question, not an issue: Has anyone encountered the `Website Unaccessible` error?

### ISSUE THREAD

**Issue by jshleeb** [matopeto/kindle-weather-dashboard#26]
Hello, thanks for sharing your project! This has been great for inspiring my own project. There isn't an issue with your repo, I'm just opening this as a way to communicate.

I'm building my own kindle dashboard webapp for the transit service in my area. When I initially access it with the kindle browser, it works and polls every minute. But after about 20min to < 2 hours of leaving the page open in the kindle browser, this error message pops up: `Website Unaccessible. Sorry, the browser cannot access this website.`. It doesn't seem to me like there would be a reason for memory usage to increase/be an issue, and this is happening on two separate kindles running `5.16.6` and `5.12.2.2`.

I'm wondering if anyone has encountered this error message before? Would you have an idea what it means, or how to work around it? Thanks!

**Comment by matopeto** [comment]
Hi



i think that `Website Unaccessible. Sorry, the browser cannot access this website.` means that the kindle could not get to the given page. Either the wifi/connection went down or the server went down. Unfortunately, I have no idea how you refresh the page in your project, where it is saved and running, but it is possible that the backend/server has some outages, or someone/something (whom the server is calling) cut off too many/frequent connections and then the whole thing crashes - but I don't know what the server that provides the data/html is exactly calling and doing.

As I wrote, I don't know how you refresh, but it would be nice to have some good error handling. In my project, the main page is loaded only once, and it is refreshed using "ajax" queries, which if they fail, nothing happens, only the page is not updated (I have not encountered this error on my project and it has been running non-stop for a year). If I were to refresh the entire website, the connection/server failure will cause the page to not be displayed at all and the next refresh will not take place

**Comment by matopeto** [comment]
I am closing this issue, but feel free to continue the discussion

**Comment by jshleeb** [comment]
Hey, thanks for replying! That's helpful to know you haven't seen this before, thanks! I also refresh using ajax queries, and when the calls fail the page also just doesn't update (different from the `Website Unaccessible` error).
